### PR TITLE
settings: Makes networking configurable.

### PR DESCRIPTION
**Pull request by abaire**

### CODE REVIEW

File: Includes/networkManager.cpp [hunk 1]
```diff
@@ -24,10 +24,12 @@ void NetworkManager::thread_main(NetworkManager *manager) {
 #ifdef NXDK
   const Config &config = manager->config;
```
Comment (dracc) [inline Includes/networkManager.cpp:25]: nit: Consider changing this to `const netConfig& config = &manager->config.settings.net;`, as no other parts of the `Config` class are used.

**Review by dracc — COMMENTED** [review]
Looks great, thanks!

File: Includes/networkManager.cpp (hunk 1 above)
Comment (abaire) [reply Includes/networkManager.cpp]: Good point, changed.

**Comment by dracc** [comment]
Thanks! :+1:

**Pull request merged**